routes/ffxiv.py

```
@@ -46,3 +46,54 @@
 @ffxiv_bp.route("/pricecheck", methods=["GET", "POST"])
 def ffxiv_pricecheck():
     return redirect("https://saddlebagexchange.com/price-sniper")
+
+
+@app.route("/petshoppinglist", methods=["GET", "POST"])
```

Ruff

Undefined name `app`

(F821)